docs/gecko.rst

```diff
@@ -38,19 +38,19 @@ Create An FTL Translation Resource
 Translations for L20n are stored in ``.ftl`` files.  You can find more 
 information about the FTL format at the end of this document.
 
-The FTL files leverage the existing packaging infrastructure for locales.  The 
-*en-US* translations for ``browser`` can be found in 
-``browser/locale/en-US/browser``, as specified by ``browser/locale/jar.mn``::
+The FTL files leverage the existing packaging infrastructure for locales.  The
+translations for ``browser`` can be found in
+``browser/locale/*/browser``, as specified by ``browser/locale/jar.mn``::
```

Comment:
Maybe rephrase like so:

    Please only package complete directory structures, and keep 
    the same directory layout as in the source.